### PR TITLE
refactor(StringWrapper): remove useless methods

### PR DESCRIPTION
**Pull request by vicb**

### CODE REVIEW

**Comment by alexeagle** [comment]
Are these useless because ts2dart translates them, or because ES6 and dart both have the methods natively and we somehow are okay at runtime using them now?

**Comment by vicb** [comment]
@alexeagle there are useless because they have the same name/signature in both ES6 & Dart (& facade as well). Not sure about "we somehow are okay at runtime using them now?" - weren't we suppose to try to remove the facade as much as possible ?

I don't get why Travis fails in router mode...

**Comment by alexeagle** [comment]
Sure, we want to remove the facade, but I'm surprised we have these methods available at runtime in the browsers we target

**Comment by vicb** [comment]
only `str.startsWith()` should be new in ES6 and the facade only forwaded the call here.

could that be an issue ? (there is still the travis error in the router mode)

File: modules/angular2/src/core/compiler/template_parser.ts [hunk 1]
```diff
@@ -254,8 +254,8 @@ class TemplateParseVisitor implements HtmlAstVisitor {
     var templateBindingsSource = null;
     if (attr.name == TEMPLATE_ATTR) {
       templateBindingsSource = attr.value;
-    } else if (StringWrapper.startsWith(attr.name, TEMPLATE_ATTR_PREFIX)) {
-      var key = StringWrapper.substring(attr.name, TEMPLATE_ATTR_PREFIX.length);  // remove the star
+    } else if (attr.name.startsWith(TEMPLATE_ATTR_PREFIX)) {
```
Comment (alexeagle) [inline modules/angular2/src/core/compiler/template_parser.ts]: still need the import in this file?

**Pull request closed without merging**